ClientGenerator/src/googleapis/codegen/languages/csharp/default/features.json

```diff
@@ -24,6 +26,7 @@
     "Google.Apis.Pubsub.v1": "Google.Cloud.PubSub.V1",
     "Google.Apis.CloudRedis.v1": "Google.Cloud.Redis.V1",
     "Google.Apis.CloudScheduler.v1": "Google.Cloud.Scheduler.V1",
+    "Google.Apis.SecretManager.v1": "Google.Cloud.SecretManager.V1",
```

Comment:
We don't have a GA version of this library yet - is that okay?

Reply:
Thanks, removed.